### PR TITLE
Auto create update SQL

### PR DESCRIPTION
**Pull request by Llewellynvdm**
Resolves gh-96 by adding the feature to auto create the update SQL. Also fixes gh-100 Field Length Issue.

### CODE REVIEW

**Comment by Llewellynvdm** [comment]
@tonypartridge you are welcome to also tryout this staging branch and give us your feedback.

File: README.md [hunk 1]
```diff
@@ -122,4 +122,4 @@ Component Builder is mapped as a component in itself on my local development env
 
 Come on buy me a coffee :)
  * PayPal: pay@vdm.io
- * Bitcoin: 3H35PYwikEosvcjxHXGPLS1GufZ4b4iFu3
\ No newline at end of file
+ * Bitcoin: 1Ga1WZyxC4ArSKuiFpUxdqzjfkAR1kJhZh
```
Comment (Llewellynvdm) [inline README.md]: A new Bitcoin wallet just for JCB was created.

File: admin/helpers/compiler/a_Get.php [hunk 2]
```diff
@@ -2152,6 +2227,210 @@ public function setGetData($ids,$view_code)
...
+	 */
+	protected function getHistoryWatch($type, $id)
+	{
+		// quick class object to store old history object
+		$this->tmpHistory = null;
+		// Create a new query object.
+		$query = $this->db->getQuery(true);
+
+		$query->select('h.*');
+		$query->from('#__ucm_history AS h');
```
Comment (Llewellynvdm) [inline admin/helpers/compiler/a_Get.php]: So we depend on the history component.

Comment (ro-ot) [inline admin/helpers/compiler/a_Get.php]: This seems more stable then the other ideas that were on the table.

**Comment by Llewellynvdm** [comment]
Yes that feature is not ready... will add it eventually. So I pushed out what I already had finished, so we can start testing it.

File: admin/helpers/compiler/c_Fields.php [hunk 3]
```diff
@@ -1026,6 +1033,93 @@ private function setField($setType, $taber, &$fieldAttributes, &$name, &$typeNam
 				$fieldSet .= PHP_EOL."\t\t\t</fields>";
 				$fieldSet .= PHP_EOL."\t\t</field>";
 			}
+			// set the repeatable field
+			elseif ($typeName === 'subform')
+			{
+				// now add to the field set
```
Comment (Llewellynvdm) [inline admin/helpers/compiler/c_Fields.php]: If Subform is correctly set as a field type in JCB it will also now work.

**Pull request merged**